stress_test_model_cb.R

```diff
@@ -455,8 +463,17 @@ for (i in seq(1, nrow(transition_scenarios))) {
       )
   }
 
+  rows_bonds <- nrow(bonds_annual_profits)
   bonds_annual_profits <- bonds_annual_profits %>%
-    left_join(financial_data_bonds, by = c("company_name", "id" = "corporate_bond_ticker", "ald_sector", "technology", "year"))
+    dplyr::inner_join(
+      financial_data_bonds,
+      by = c("company_name", "id" = "corporate_bond_ticker", "ald_sector", "technology")
+    )
+  cat("number of rows dropped from loan book by joining financial data on
```

Comment:
NP: note that  n rows could also increase after an inner join. However this will soon not be possible anymore once duplicates checks and filterings are applied to all datasets.